domains/transfer/Transfer.API/API/ApiModels/Requests/CreateTransferAgreementValidator.cs

```diff
@@ -33,5 +33,10 @@ public CreateTransferAgreementValidator(IHttpContextAccessor context)
             .WithMessage("ReceiverTin must be 8 digits without any spaces.")
             .NotEqual(context.HttpContext.User.FindSubjectTinClaim())
             .WithMessage("ReceiverTin cannot be the same as SenderTin.");
+
+        RuleFor(createTransferAgreement => createTransferAgreement.Base64EncodedWalletDepositEndpoint)
+            .NotEmpty()
+            .WithMessage("Base64EncodedWalletDepositEndpoint cannot be empty.")
+            .MustBeValidBase64("Base64-encoded Wallet Deposit Endpoint is not valid");
```

Comment:
This only ensures that the input is correctly base64 encoded. But as far as I can see, there is no validation or check if the decoded content can be correctly JSON deserialized.

Reply:
Fixed:

https://github.com/Energinet-DataHub/energy-origin/pull/443/commits/878950fe07707c18824427a42d7843b167a05c27